ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/dropdown.html

```diff
@@ -33,31 +33,65 @@
-            data-sly-attribute.multiple="${dropdown.isMultiSelect ? 'multiple' : ''}"
-            aria-readonly="${dropdown.readOnly ? 'true' : ''}">
-        <div data-sly-test.isPlaceHolderSet="${dropdown.placeHolder != null}" data-sly-unwrap></div>
-        <option data-sly-test="${!isPlaceHolderSet}" disabled selected></option>
-        <option data-sly-test="${isPlaceHolderSet}" value="" disabled selected>${dropdown.placeHolder}</option>
-        <div data-sly-list.enumName="${dropdown.enumNames}" data-sly-unwrap>
-            <div data-sly-test="${enumName != null && dropdown.enums[enumNameList.index] != null}" data-sly-unwrap>
-                <option value="${dropdown.enums[enumNameList.index].toString}"
-                        class="cmp-adaptiveform-dropdown__option"
-                        data-sly-attribute.selected="${dropdown.enums[enumNameList.index] in dropdown.default ? 'selected' : ''}">${enumName}</option>
+
+    <div data-sly-test="${!dropdown.properties.fd:autoComplete}" data-sly-unwrap>
+        <select class="cmp-adaptiveform-dropdown__widget"
+                id="${widgetId}"
+                title="${dropdown.tooltipVisible ? '' : dropdown.tooltipText}"
+                name="${dropdown.name}"
+                data-cmp-data-layer="${dropdown.data.json}"
+                disabled="${!dropdown.enabled || dropdown.readOnly}"
+                required="${dropdown.required}"
+                data-sly-attribute.multiple="${dropdown.isMultiSelect ? 'multiple' : ''}"
+                aria-readonly="${dropdown.readOnly ? 'true' : ''}">
+            <div data-sly-test.isPlaceHolderSet="${dropdown.placeHolder != null}" data-sly-unwrap></div>
+            <option data-sly-test="${!isPlaceHolderSet}" disabled selected></option>
+            <option data-sly-test="${isPlaceHolderSet}" value="" disabled selected>${dropdown.placeHolder}</option>
+            <div data-sly-list.enumName="${dropdown.enumNames}" data-sly-unwrap>
+                <div data-sly-test="${enumName != null && dropdown.enums[enumNameList.index] != null}"
+                     data-sly-unwrap>
+                    <option value="${dropdown.enums[enumNameList.index].toString}"
+                            class="cmp-adaptiveform-dropdown__option"
+                            data-sly-attribute.selected="${dropdown.enums[enumNameList.index] in dropdown.default ? 'selected' : ''}">
+                        ${enumName}
+                    </option>
+                </div>
+            </div>
+        </select>
+    </div>
+    <div data-sly-test="${dropdown.properties.fd:autoComplete}" data-sly-unwrap>
+        <datalist id="${widgetId}">
+            <option data-sly-test="${!dropdown.placeHolder}" disabled selected></option>
+            <option data-sly-test="${dropdown.placeHolder}" value="" disabled selected>${dropdown.placeHolder}</option>
+
+            <div data-sly-list.enumName="${dropdown.enumNames}" data-sly-unwrap>
```

Comment:
place holder logic which existed does not seem to be implemented for auto-complete